src/Microsoft.Data.Analysis/StringDataFrameColumn.cs

```diff
@@ -169,10 +169,10 @@ public IEnumerator<string> GetEnumerator()
 
         public override DataFrameColumn Filter<U>(U lower, U upper) => throw new NotSupportedException();
 
-        public new StringDataFrameColumn Sort(bool ascending = true)
+        public new StringDataFrameColumn OrderBy()
         {
             PrimitiveDataFrameColumn<long> columnSortIndices = GetAscendingSortIndices();
-            return Clone(columnSortIndices, !ascending, NullCount);
+            return Clone(columnSortIndices, false, NullCount);
```

Comment:
Can you use a named parameter for the hard-coded boolean? It makes the code a lot easier to read/understand:

```C#
return Clone(columnSortIndices, invertMapIndices: false, NullCount);
```

vs.

```C#
return Clone(columnSortIndices, false, NullCount);
```